lametro/models.py

```diff
@@ -443,32 +444,19 @@ def ceo(cls):
 
     @property
     def headshot_url(self):
-        if self.headshot:
-            # Assigning this to image_url would make 'return static(image_url)'
-            # at the end of this property concatenate 'static' to the url.
-            # Returning here solves that.
-            if "headshot_placeholder.png" not in self.headshot.path:
-                return self.headshot
-
-        file_directory = os.path.dirname(__file__)
-        absolute_file_directory = os.path.abspath(file_directory)
-
-        filename = self.slug_name + ".jpg"
-
-        manual_headshot = os.path.join(
-            absolute_file_directory, "static", "images", "manual-headshots", filename
-        )
-
-        if Path(manual_headshot).exists():
-            image_url = "images/manual-headshots/" + filename
-
-        elif self.headshot:
-            image_url = self.headshot.url
-
-        else:
-            image_url = "images/headshot_placeholder.png"
+        """
+        Board member headshots are stored in an S3 bucket (see
+        councilmatic/custom_storages.py). Each board member should
+        have a matching headshot in the bucket named <board-member-slug>.jpg.
+        """
 
-        return static(image_url)
+        try:
+            media_storage = MediaStorage()
+            headshot_file = f"{self.slug_name}.jpg"
+            return media_storage.url(headshot_file)
```

Comment:
also, is there something that's changing the name of the file to this format when it's uploaded? i'm imagining an instance where they've uploaded png instead of a jpg, or if the name of the file itself is different